sdk/storage/storage-queue/src/RetryPolicyFactory.ts

```diff
@@ -2,24 +2,30 @@
 // Licensed under the MIT License.
 
 import { RequestPolicy, RequestPolicyFactory, RequestPolicyOptions } from "@azure/core-http";
-import { RetryPolicy, RetryPolicyType } from "./policies/RetryPolicy";
+import {
+  RetryPolicy as StorageQueueRetryPolicy,
+  RetryPolicyType as StorageQueueRetryPolicyType
+} from "./policies/RetryPolicy";
 
-export { RetryPolicyType } from "./policies/RetryPolicy";
+export {
+  RetryPolicyType as StorageQueueRetryPolicyType,
+  RetryPolicy as StorageQueueRetryPolicy
+} from "./policies/RetryPolicy";
 
 /**
  * Retry options interface.
```

Comment:
Is the below helpful? Am not sure. Just putting the idea out there :)

```suggestion
 * Retry options interface. This interface is very specific to how retries work in Storage libraries.
 * Do not try to re-use them when working with other libraries.
```

Reply:
Making rename changes in separate PR